### PR TITLE
Feature/SSCS-2522-add-a11y-test

### PR DESCRIPTION
**Pull request by vianvi**
*Added accessibility stage to pipeline

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -157,35 +158,83 @@ lookAndFeel.configure(app, {
...
-  session: {
-    redis: {
-      url: config.redis.url,
-      connect_timeout: 15000
+function isTest() {
+  return process.env.NODE_ENV === 'mocha';
+}
+
+
+if (isTest()) {
```
Comment (bendiggle) [inline app.js]: I think we can put this into pa11y's own app.js. It's been done for TYA I believe and ES so should 't be too difficult. I'll have a look when I have time.

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,4 @@ functional-output/
 dist
 submit-your-appeal.iml
 mochawesome-report/
+yarn.lock
```
Comment (bendiggle) [inline .gitignore]: shouldn't ignore this. The whole point of yarn.lock is that we all have the same version

Comment (bendiggle) [inline .gitignore]: shouldn't ignore this. The whole point of yarn.lock is that we all have the same version

File: .gitignore [hunk 3]
```diff
@@ -12,3 +12,5 @@ smoke-output/
 functional-output/
 dist
 submit-your-appeal.iml
+mochawesome-report/
+yarn.lock
```
Comment (bendiggle) [inline .gitignore]: shouldn't ignore this

File: config/mocha.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      "maxAge": 2592000,
+      "sha256s": ["TODO", "Set proper SHA256s"]
+    }
+  },
+  "redis": {
+    "url": "redis://localhost:6379",
+    "secret": "defaultsecret",
+    "useSSL": "false"
+  }
+}
```
Comment (bendiggle) [inline config/mocha.json]: what's this for?

Comment (vianvi) [reply config/mocha.json]: to make redis available on CNP

File: .gitignore (hunk 3 above)
Comment (vianvi) [reply .gitignore]: done

File: config/mocha.json (hunk 4 above)
Comment (vianvi) [reply config/mocha.json]: to make redis available on CNP

**Pull request merged**